modules/phase_field/doc/content/source/auxkernels/PFFractureBoundsAux.md

```diff
@@ -0,0 +1,28 @@
+# PFFractureBoundsAux
+
+!syntax description /AuxKernels/PFFractureBoundsAux
+
+## Description
+
+`PFFractureBoundsAux` is used for setting bounds on phase field fracture variable. The upper bound is set as 1 by default. The lower bound is set as variable's old value in order to enforce irreversibility condition.
+
+Note that in order for these bounds to have an effect, the user has to specify the
+PETSc options `-snes_type vinewtonssls` or `-snes_type vinewtonrsls`. Both
+options emply semi-smooth non-linear solution algorithms to solve the
+variational inequality. However, the `vinewtonrsls` option employs a reduced
+space active set method that actually reduces the size of the linear system by
```

Comment:
`vinewtonrsls`  solves a reduced linear system of equations, corresponding to the inactive set, to calculate a direction. The solution regarding the inactive set will be updated based on the new direction, while the solution on the  active set will not be changed.